ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -310,19 +313,31 @@ TorchCore::validateOutputTensor (const at::Tensor output, unsigned int idx)
   at::Tensor sliced_output = output.slice (0);
   c10::IntArrayRef sliced_output_sizes = sliced_output.sizes ();
 
+  /** if idx is in bounds */
+  if (outputTensorMeta.num_tensors <= idx) {
+    ml_loge ("Invalid output meta: trying to access index %d with total %d tensors. "
```

Comment:
For users grepping error messages, a string is not supposed to be line-broken. (even in 80col-strict Linux communities). Please keep the string in a single double-quote pair.

Reply:
done. I wonder if there are code search tools to extend searching into strings/messages broken into different lines?

Reply:
We still have a lot of people using grep. :)